### PR TITLE
End of flu season crowdcast alert

### PR DESCRIPTION
**Pull request by mariajahja**

### CODE REVIEW

**Review by korlaxxalrok — APPROVED** [review]
LGTM :shipit:

File: src/epicast/epicast_emails.py [hunk 1]
```diff
@@ -45,21 +45,78 @@ class Template:
 
     # placeholder for one-off emails sent on special occasions
     ALERT = {
-      'subject': 'Crowdcast Needs Your Help',
+      'subject': 'Thank You',
       'text': '''
-        Dear %s,
```
Comment (undefx) [inline src/epicast/epicast_emails.py:50]: Hey just a quick heads-up: I didn't see a corresponding change where the string format arguments are passed in. If the username argument is still passed, but the format string doesn't have a placeholder, then I think there will be a crash. If I missed the update, then pls disregard.

Where the username is combined with this format string: https://github.com/cmu-delphi/flu-contest/blob/main/src/epicast/epicast_emails.py#L266

Comment (undefx) [reply src/epicast/epicast_emails.py:50]: Hah, you already fixed it in #12! Nevermind :)

Comment (mariajahja) [reply src/epicast/epicast_emails.py:50]: Yeah, I missed that! Still trying to figure all this out. Thanks for checking :)

**Pull request merged**